tardis/spectrum/formal_integral/tests/test_source_function.py

```diff
@@ -24,12 +24,12 @@ def source_function_verysimple(config_verysimple, atomic_dataset):
     transport = sim.transport
 
     formal_integrator = FormalIntegrator(sim_state, plasma, transport)
-    sourceFunction = SourceFunctionSolver(line_interaction_type = formal_integrator.transport.line_interaction_type)
+    sourceFunction = SourceFunctionSolver(formal_integrator.transport.line_interaction_type, 
+                                          formal_integrator.atomic_data)
```

Comment:
Is that the correct atomic_data input? In the line below that was removed you used `formal_integrator.plasma.atomic_data`.